python-checks/src/main/java/org/sonar/python/checks/regex/UnusedGroupNamesCheck.java

```diff
@@ -117,31 +118,51 @@ private void checkGroupAccesses(RegexParseResult regexParseResult, KnownGroupsCo
   }
 
   private void checkGroupNameOrNumberExists(RegexParseResult regexParseResult, KnownGroupsCollector groupsCollector, Expression argumentExpression) {
```

Comment:
This method name is no longer accurate, I suggest renaming to `checkValidGroupNameAccess`

Reply:
done